### PR TITLE
feat(releases): NPM release van JSON tokens beschikbaar maken [Lux-247]

### PR DESCRIPTION
**Pull request by MrSkippy**
Creates a package with JSON-tokens (and CSS from `dist/`)
Added Readme to the package

### CODE REVIEW

File: lerna.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
   "command": {
     "publish": {},
     "version": {
-      "allowBranch": ["main"],
+      "allowBranch": ["main", "LUX-247_(NPM)-Release-van-JSON-tokens-beschikbaar-maken"],
```
Comment (MrSkippy) [inline lerna.json:9]: TODO for @MrSkippy : remove the feature branch again

File: proprietary/design-tokens/README.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+Daarna zijn de volgende tokens beschikbaar:
+
+## JSON
+
+In de map `/lib/tokens/` staan de tokens die vanuit Tokens Studio en Figma gegenereerd worden.
+Deze tokens kunnen gebruikt worden in combinatie met je eigen tokens of om je eigen variabelen te bouwen.  
+_Let op_: Sommige mapnamen bevatten nog `(beta)`, deze suffix zal in de toekomst verdwijnen, dus het wordt aangeraden deze uit je output te filteren.  
+In de source code op Github kan je onze build script(s) inzien, deze worden niet meegeleverd in de package aangezien ze specifiek voor het LUX-project zijn.
+
+Meer informatie over de LUX design tokens en de lagen vind je op de pagina [Lagen van Design Tokens](https://nl-design-system.github.io/lux/?path=TODO:include-link) van onze Storybook.
```
Comment (MrSkippy) [inline proprietary/design-tokens/README.md:25]: TODO @MrSkippy : include correct link

File: .github/workflows/continuous-deployment.yml [hunk 3]
```diff
@@ -169,7 +169,7 @@ jobs:
   publish-npm:
     runs-on: ubuntu-latest
     needs: [lint, test]
-    if: github.ref == 'refs/heads/main'
+    if: github.ref == 'refs/heads/main' || 'refs/heads/LUX-247_(NPM)-Release-van-JSON-tokens-beschikbaar-maken'
```
Comment (MrSkippy) [inline .github/workflows/continuous-deployment.yml:172]: TODO @MrSkippy : remove branch

File: proprietary/design-tokens/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Design Tokens voor het LUX Design System
+
+Deze package bevat de Design Tokens van het LUX Design System in verschillende formaten.  
+In design tokens worden de keuzes opgeslagen voor het design system. De design tokens in deze package zijn de enige bron van waarheid voor de look en feel van de componenten.
```
Comment (remypar5) [inline proprietary/design-tokens/README.md:4]:
```suggestion
In design tokens worden de design beslissingen vastgelegd voor het design system. De design tokens in deze package zijn de enige bron van waarheid voor de look en feel van de componenten.
```

File: proprietary/design-tokens/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+
+`npm install @lux-design-system/design-tokens --save-dev`
+
+### Yarn
+
+`yarn add @lux-design-system/design-tokens --dev`
+
+Daarna zijn de volgende tokens beschikbaar:
+
+## JSON
```
Comment (remypar5) [inline proprietary/design-tokens/README.md:18]:
```suggestion
## PNPM

`pnpm add -D @lux-design-system/design-tokens`

## JSON
```

**Review by remypar5 — CHANGES_REQUESTED** [review]

**Review by remypar5 — APPROVED** [review]

**Pull request merged**